src/utils/image_utils.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import cv2
+
+
+# Converts frame to grayscale
+def convert_to_grayscale(frame):
+    return cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+
+
+# Function to find the absolute difference between two frames
+def absolute_difference(curr_frame, prev_frame):
+    curr_frame_grayscale = convert_to_grayscale(curr_frame)
+    prev_frame_grayscale = convert_to_grayscale(prev_frame)
+    frame_diff = cv2.absdiff(curr_frame_grayscale, prev_frame_grayscale)
+    return frame_diff.sum()
+
+
+# Function to find the mean squared error between two frames
+def mse_difference(curr_frame, prev_frame):
+    curr_frame_gray = convert_to_grayscale(curr_frame)
+    prev_frame_gray = convert_to_grayscale(prev_frame)
+    total_pixels = curr_frame_gray.shape[0] * curr_frame_gray.shape[1]
+    frame_diff = curr_frame_gray.astype('float') - prev_frame_gray
+    mse_diff = np.sum(frame_diff**2) / float(total_pixels)
+    return mse_diff
+
+
+# Function to blacken out background of frame
+def mask_background(frame, dim=21, threshold1=10, threshold2=200, 
```

Comment:
Let us keep parameter annotation a standard for our system.

Reply:
Added